src/components/analytics/GoogleAnalyticsProvider.tsx

```diff
@@ -1,12 +1,13 @@
-import ReactGA from 'react-ga4'
+/* eslint-disable no-unused-vars */
+
 import { GaOptions, InitOptions, UaEventOptions } from 'react-ga4/types/ga4'
 
 /**
  * Google Analytics Provider containing all methods used throughout app to log events to Google Analytics.
  */
 export default class GoogleAnalyticsProvider {
   public sendEvent(event: string | UaEventOptions, params?: any) {
-    ReactGA.event(event, params)
+    // ReactGA.event(event, params)
```

Comment:
Why are you commenting these out, instead of just removing them from the codebase?